packages/output-view/README.md

```diff
@@ -0,0 +1,6 @@
+# Theia - Output View Extension
+
+See [here](https://github.com/theia-ide/theia) for a detailed documentation.
```

Comment:
please add two sentences about what the output extension does.

Reply:
Done